.github/workflows/release.yml

```diff
@@ -48,14 +48,14 @@ jobs:
       - name: Install Unity engine
         uses: ./.github/.release/actions/actions/installers/unity
         with:
-          unity-version: "2020.3.22f1"
+          unity-version: "2022.3.11f1"
```

Comment:
@parfeon  says on GH we have `Unity Editor version:    2020.3.22f1 (e1a7f79fd887)`, but new changes going with 11f1?

Reply:
If we want to go with `2022.3.11f1`, then should we have a job or step in yml file to install the version we want?

Reply:
@mohitpubnub this won't work because hash is not in the list which I shared

Reply:
@mohitpubnub I'll need some time to add more versions